app/jobs/sync_salesforce_opportunities_job.rb

```diff
@@ -7,7 +7,7 @@ def perform(contact_id=nil)
     if contact_id
       sf_opportunities = OpenStax::Salesforce::Remote::Opportunity.where(contact_id:contact_id)
     else
-      sf_opportunities = OpenStax::Salesforce::Remote::Opportunity.all
+      sf_opportunities = OpenStax::Salesforce::Remote::Opportunity.where(record_type_name:'Book Opp')
```

Comment:
If you want to, you can put this in `openstax_salesforce`, so there is only one place this needs to be updated if it ever changes?
I think you could add a method, so you could do something like `OpenStax::Salesforce::Remote::Opportunity.book_opportunities` and filter the query there.

Alternatively, we could have a class for each type of opportunity (like we kind of do for a Salesforce Account - we have school but there are also partners, for eg).

[Here is a code example](https://github.com/openstax/openstax_salesforce/blob/master/lib/openstax/salesforce/remote/school.rb#L15-L17)

Up to you though.

Reply:
I tried to get this to work, but it is failing. I'll leave it as is for now.